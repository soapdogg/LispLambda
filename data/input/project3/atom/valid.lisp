(ATOM 4)
(ATOM (ATOM 45))
(ATOM (CONS 2 9))
(ATOM (ATOM (ATOM T)))
(ATOM (QUOTE (7 10)))
(ATOM (QUOTE ZZ))