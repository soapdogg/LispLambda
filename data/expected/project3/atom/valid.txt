T
T
NIL
T
NIL
T